### PR TITLE
PR to Install Rancher Desktop and push Epinio

### PR DESCRIPTION
**Pull request by mmartin24**
Implementation of related issue: https://github.com/epinio/epinio/issues/940

Recap:

- KVM needs to be enabled on runners to install RD
- Nightly CI should test Epinio against latest release of Rancher Desktop.
- use rdctl to install RD
- install epinio
- check if epinio endpoint can be reached
- push sample app
- check if sample app can be reached

### CODE REVIEW

**Comment by mmartin24** [comment]
Attempted to do first draft of Github Action installing Rancher Desktop on an Ubuntu Image.
Waiting for OpenVPN access to check the full flow on test runner, but meanwhile I tried on a VM with kvm enabled and I have issues after installing Rancher Desktop. 

I cannot use `rdctl start` or `rdctl set --container-engine docker --kubernetes-version 1.21.2` from (https://docs.rancherdesktop.io/references/rdctl-command-reference) .
I can execute thought `rancher-desktop` on terminal and windows will pop, but that is not the intended way here, so I am not sure how to proceed directly from terminal.
image

@nunix, I am not sure if you'd know a way?

**Comment by mmartin24** [comment]
SInce I still cannot test in GitHub runner I created a VM with Ubuntu 22.04 and built this script ([rtest.sh](.github/workflows/rdtest.sh) ) that deploys Rancher Desktop from scratch, opens it, changes configuration to Docker and later deploys Epinio. 

It still needs polish but as can be seen in the screenshot does the basic part. Once fully functional, I will try to pass it to Github Actions and test in runner:
image
Thanks @nunix for the help

**Comment by mmartin24** [comment]
> I cannot use `rdctl start` or `rdctl set --container-engine docker --kubernetes-version 1.21.2` from (https://docs.rancherdesktop.io/references/rdctl-command-reference) . I can execute thought `rancher-desktop` on terminal and windows will pop, but that is not the intended way here, so I am not sure how to proceed directly from terminal. Here a screenshot of this on an openSUSE TW virtual machine image
> 

This issue may be a bug. Opened this for taking a look: https://github.com/rancher-sandbox/rancher-desktop/issues/2683

**Comment by mmartin24** [comment]
I attempted to replicate what I had on the [rdtest.sh](https://github.com/epinio/epinio/pull/1674/commits/2ff1363bc53209bdface6e191010e50903fafb23) script into Github actions. For that I improved a bit the `rancher-desktop.yml` and also created a new `scenario7_test.go` reusing the exact parts of other tests checking for  IP, Installing Epinio, connecting and pushing an app with configuration. The `rancher-desktop.yml` file is based on the `rke.yml` as it intends to point to a self-hosted runner.

However I am quite new at GitHub Actions and I am not sure I had the script correctly set. In nutshell I want to download Rancher Desktop using an Ubuntu 22.04 image using a self hosted runner. I would appreciate if someone could take a look and provide me some feedback. I will be happy to meet if needed.

File: acceptance/install/scenario7_test.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+		flags             []string
+		epinioHelper      epinio.Epinio
+		configurationName = catalog.NewConfigurationName()
+		appName           string
+		loadbalancer      string
+		registryUsername  string
+		registryPassword  string
+		// rangeIP           string
+		domain            string
+		domainIP          string
```
Comment (mmartin24) [inline acceptance/install/scenario7_test.go:29]: this fails both when is commented and when it is not.
As I undestand it `domainIP` is being used later in line 45 `domainIP = strings.TrimSuffix(domain, ".omg.howdoi.website")`; however I get [this](https://github.com/epinio/epinio/runs/7766953484?check_suite_focus=true) error:

```
# github.com/epinio/epinio/acceptance/install_test
Error: vet: acceptance/install/scenario7_test.go:29:3: domainIP declared but not used
```
I am not sure why. @enrichman would you know?

Comment (mmartin24) [reply acceptance/install/scenario7_test.go]: Spoke offline about it.
I am commenting both lines out

**Comment by mmartin24** [comment]
I tried to run this pr in the `RKE-CI` runner and for some reason is launching the `acceptance-scenario3` instead of  `acceptance-scenario7`: https://github.com/epinio/epinio/runs/7769764312?check_suite_focus=true

@svollath , do spot what am I missing?

**Comment by svollath** [comment]
You just scheduled the RKE-CI, and therefore scenario3 was used, on a random runner. I think in order to test your workflow, you need to push your PR, so that “Rancher Desktop Install” becomes available for scheduling.

**Comment by mmartin24** [comment]
> You just scheduled the RKE-CI, and therefore scenario3 was used, on a random runner. I think in order to test your workflow, you need to merge your PR, so that “Rancher Desktop Install” becomes available for scheduling.

Thanks, I am not sure if it is ok to do a merge to try if it works. I think I will try to split this ticket into smaller ones and test on a separate environment the self-hosted runner functioning.

**Comment by mmartin24** [comment]
Split into smaller  tasks #1682 #1683 and #1684 .
Closing this.

**Pull request closed without merging**